### ISSUE TITLE
Generate embeddings

### ISSUE THREAD

**Issue by christopher5106** [facebookresearch/XLM#86]
Hi,

In the embedding generation example you give:
https://github.com/facebookresearch/XLM/blob/master/generate-embeddings.ipynb
the model is in training mode, which is not good because of the presence of dropout layers... 
I think you should add 
model.training = False
or something equivalent ?

**Comment by glample** [comment]
Hi,

You are right. `model.training = False` will not work, as it won't impact the internal modules. Instead you can do `model.eval()` after creating the model.

**Comment by glample** [comment]
Fixed in https://github.com/facebookresearch/XLM/commit/03d7e94e04abeb238a4b7b76d78417e96016cba1
